pages/borrow/[symbol].tsx

```diff
@@ -61,13 +65,146 @@ const Borrow: NextPage<
     () => parseUnits(collateralValue, collateral?.decimals ?? 18),
     [collateralValue, collateral?.decimals],
   )
+  const collateralBalance = useMemo(
+    () => (collateral ? balances[collateral.address] : 0n),
+    [balances, collateral],
+  )
 
   const loanAmount = useMemo(
     () => parseUnits(loanValue, asset.underlying.decimals),
     [loanValue, asset.underlying.decimals],
   )
 
-  console.log(collateralAmount, loanAmount)
+  const maxLiquidationTargetLtv = useMemo(
+    () =>
+      collateral
```

Comment:
Let's make `collateral` a `Collateral` instead of a `Currency`.

Reply:
@onemonster, I think Currency is fine because we already use the <CurrencySelect> component in this file.